### PR TITLE
Remove unused variable and move reset session to a function

### PR DESCRIPTION
**Pull request by k7z45**
Per https://github.com/web-platform-tests/wpt/pull/27734/files/1abbaacbe6091d252040e6a5405e4ee90b3a64f8#r583473293
move duplicate code in session and bidi_session fixture into its own function and removed unused variable is_cur_bidi

### CODE REVIEW

File: webdriver/tests/support/fixtures.py [hunk 1]
```diff
@@ -123,6 +123,21 @@ def configuration():
     }
 
 
+async def reset_current_session_if_necessary(caps, bidi):
```
Comment (whimboo) [inline webdriver/tests/support/fixtures.py:126]: Maybe we could rename the argument to `request_bidi`?

**Review by whimboo — APPROVED** [review]
Looks fine to me. Thanks! Just a small nit regarding naming of the argument.

File: webdriver/tests/support/fixtures.py (hunk 1 above)
Comment (k7z45) [reply webdriver/tests/support/fixtures.py]: Done, would you be able to merge this? Thanks!

**Comment by whimboo** [comment]
No, I'm not able to merge PRs. @jgraham would you mind doing that?

**Comment by k7z45** [comment]
@stephenmcgruer 

Hi Stephen,

Could you help merging this PR, please?

**Pull request merged**